src/dialog/src/Dialog.js

```diff
@@ -209,94 +217,91 @@ class Dialog extends React.Component {
       minHeightContent
     } = this.props
 
-    let maxHeight
-    if (Number.isInteger(topOffset)) {
-      maxHeight = `calc(100vh - ${topOffset}px)`
-    } else {
-      maxHeight = `calc(100vh - ${topOffset})`
-    }
+    const topOffsetWithUnit = Number.isInteger(topOffset)
+      ? `${topOffset}px`
```

Comment:
I don't think we need a unit here

Reply:
We do need this one for the `maxHeight` `calc()` below.

Reply:
`${topOffset}px` => `topOffset` right?

Reply:
No, because then `calc(100% - ${topOffsetWithUnit} * 2)` won't work.